sdk/turing/router/config/router_config.py

```diff
@@ -216,20 +220,40 @@ def to_open_api(self) -> OpenApiModel:
             kwargs['enricher'] = self.enricher.to_open_api()
         if self.ensembler is not None:
             kwargs['ensembler'] = self.ensembler.to_open_api()
+            if kwargs['ensembler'] is None:
+                # Nop ensembler, delete
+                del kwargs['ensembler']
 
         return turing.generated.models.RouterConfig(
             environment_name=self.environment_name,
             name=self.name,
             config=turing.generated.models.RouterVersionConfig(
                 routes=[route.to_open_api() for route in self.routes],
-                default_route_id=self.default_route_id,
                 experiment_engine=self.experiment_engine.to_open_api(),
                 timeout=self.timeout,
                 log_config=self.log_config.to_open_api(),
                 **kwargs
             )
         )
 
+    def _get_default_route_id(self):
+        default_route_id = self.default_route_id
+        # If nop config is set, use the final_response_route_id as the default
+        if (self.ensembler.type == "nop" and
+            self.ensembler.nop_config is not None and
+            self.ensembler.nop_config.final_response_route_id is not None):
+            default_route_id = self.ensembler.nop_config.final_response_route_id
+        self._verify_default_route_exists(default_route_id)
+        return default_route_id
+
+    def _verify_default_route_exists(self, default_route_id):
+        for route in self.routes:
+            if route.id == default_route_id:
+                return
+        raise turing.router.config.route.InvalidRouteException(
```

Comment:
Since we're already importing `Route` via `from turing.router.config.route import Route`, perhaps we could just import `InvalidRouteException` and use it here too?